libs/src/api/Track.ts

```diff
@@ -582,7 +583,8 @@ export class Track extends Base {
       metadataMultihash,
       metadataFileUUID,
       transcodedTrackCID,
-      transcodedTrackUUID
+      transcodedTrackUUID,
+      metadata: respMetadata
```

Comment:
what's the difference betwen respMetadata and the metadata we upload? not sure why we need to return this in libs either

Reply:
yeah so initially i tried passing `metadata` (the param here) straight to `addTracksToChainAndCnode`. however streaming didn't work doing that because here `this.creatorNode.uploadTrackContent` uploads the track file and sets various related metadata fields like `track_segments` that we still rely on. the difference between the `metadata` param and `respMetadata` is these trackfile-related fields like `track_segments`.